independent-publisher-connectors/OMDb API/apiDefinition.swagger.json

```diff
@@ -0,0 +1,167 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "The OMDb API is a RESTful web service to obtain movie information, all content and images on the site are contributed and maintained by our users.\nThis API requires authorization, you can get a free key here: [http://omdbapi.com/apikey.aspx](http://omdbapi.com/apikey.aspx)",
+    "version": "1.0",
+    "title": "OMDb",
+    "termsOfService": "http://omdbapi.com/legal.htm",
+    "contact": {
```

Comment:
The contact section should include a "url" : "the product website url"
Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json)

Reply:
Done